clang/lib/DPCT/SrcAPI/APINames.inc

```diff
@@ -1679,7 +1679,7 @@ ENTRY(cuMemHostUnregister, cuMemHostUnregister, true, NO_FLAG, P4, "DPCT1026/DPC
 ENTRY(cuMemcpy, cuMemcpy, true, NO_FLAG, P4, "Successful")
 ENTRY(cuMemcpy2D, cuMemcpy2D_v2, true, NO_FLAG, P4, "Successful")
 ENTRY(cuMemcpy2DAsync, cuMemcpy2DAsync_v2, true, NO_FLAG, P4, "Successful")
-ENTRY(cuMemcpy2DUnaligned, cuMemcpy2DUnaligned_v2, false, NO_FLAG, P4, "comment")
+ENTRY(cuMemcpy2DUnaligned, cuMemcpy2DUnaligned_v2, true, NO_FLAG, P4, "Successful")
```

Comment:
need add lit test for cuMemcpy2DUnaligned

Reply:
Reverted the latest commit that removed the lit tests for `cuMemcpy2DUnaligned`